### DISCUSSION TITLE
Image optimisation

### DISCUSSION THREAD

**Discussion by stephenmeehanuk** [statamic/cms#6476] [General]
Hi,

Are there any plans to add image optimisation to Statamic? 

I usually [ImageOptim](https://imageoptim.com/mac) locally, but I don't think any images generated by Statamic/Glide are optimised in the same way - are they?

I can see there's a [plugin](https://statamic.com/addons/4rn0/imageoptimizer-v3) that does this just this, are there any plans to bring similar functionality into core?

**Comment by jackmcdade** [comment]
Optimization can be done with the quality parameter.

 quality=“50”

**Reply by stephenmeehanuk** [reply]
Ah, great.

Behind the scenes is Statamic using doing something similar to ImageOptim?

**Reply by jasonvarga** [reply]
It'll be GD's or Imagemagick's (whichever you have configured) quality encoding that does it.

**Comment by jackmcdade** [comment]
Just so you know though, it’s not quite as good as ImageOptim.
